_maps/shuttles/aux_base_default.dmm

```diff
@@ -44,10 +44,6 @@
 /obj/effect/turf_decal/stripes/line,
 /turf/open/floor/plating,
 /area/shuttle/auxillary_base)
-"y" = (
```

Comment:
not sure what happened here I will check tonight

Reply:
Probably fine, got mapmerged, judging by the whitespace the last person who touched this used FastDMM but didn't mapmerge it.

Reply:
blame goes to @ninjanomnom :v